ConnectorAutocadCivil/ConnectorAutocadCivil/Utils.cs

```diff
@@ -735,9 +847,9 @@ public static void SetStyle(Base styleBase, Entity entity, Dictionary<string, Ob
 
     if (styleBase["linetype"] is string lineType)
     {
-      if (lineTypeDictionary.ContainsKey(lineType))
+      if (lineTypeDictionary.TryGetValue(lineType, out ObjectId value))
```

Comment:
What happens if we don't manage to parse this? will entity.LinetypeId be set to anything useful/correct?

Reply:
It will end up using the default linetype, which would be expected behavior